cell-painting/1.optimize/get_best_hyperparameters.ipynb

```diff
@@ -475,7 +322,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.11"
+   "version": "3.7.8"
```

Comment:
I'm wondering why the version changed. We have a conda environment in this repo right?